utils/RunONNXModel.py

```diff
@@ -646,14 +652,19 @@ def main():
             end = time.perf_counter()
             elapsed = end - start
             perf_results += [elapsed]
-            print("  {} iteration: {} seconds".format(ordinal(i+1), elapsed))
+            print("  {} iteration, {}, seconds".format(ordinal(i+1), elapsed))
```

Comment:
added `,` to generates CSV files that can be scanned by Pandas without further preprocessing.